json_annotation/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 0.2.7
+
+* Added `JsonSerializable.disallowUnrecognizedKeys`.
+  * Defaults to `false` which maintains the previous behavior.
+  * Throws an `UnrecognizedKeysException` if it finds unrecognized keys in the
```

Comment:
this and next bullet: Put runtime notes in json_serialiable changelog

Reply:
done